### ISSUE TITLE
Test Failure: com.ibm.ws.logging.fat.FATSuite.testMaxFileSizePriorityRolling

### ISSUE THREAD

**Issue by fmhwong** [OpenLiberty/open-liberty#23446]
RTC 292401

```
testMaxFileSizePriorityRolling:junit.framework.AssertionFailedError: Timeout occurred. Please note the time in the report does not reflect the time until the timeout.
    at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
    at com.ibm.aries.buildtasks.Try.execute(Try.java:119)
    at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
    at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
    at com.ibm.aries.buildtasks.ForEach.execute(ForEach.java:158)
    at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
    at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
    at com.ibm.aries.buildtasks.If.execute(If.java:85)
    at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
    at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
    at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
    at com.ibm.aries.buildtasks.BuildProjects.doExecute(BuildProjects.java:399)
    at com.ibm.aries.buildtasks.BuildProjects.execute(BuildProjects.java:231)
    at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
    at com.ibm.aries.buildtasks.Iff$Clause.execute(Iff.java:82)
    at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
    at com.ibm.aries.buildtasks.Iff.execute(Iff.java:130)
    at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
    at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
    at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
    at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
    at com.ibm.aries.buildtasks.ForEach.execute(ForEach.java:158)
    at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
    at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
    at com.ibm.aries.buildtasks.Try.execute(Try.java:119)
    at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
    at com.ibm.aries.buildtasks.If.execute(If.java:85)
    at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
```

This only occurred on Java 18 and 19 builds.

**Comment by fmhwong** [comment]
@ellen-lau This happens quite frequently.  Can you take a look?

**Comment by ellen-lau** [comment]
This should have been addressed by: #24686, closing.
